projects/views.py

```diff
@@ -155,3 +158,26 @@ def get(self, request, *args, **kwargs):
 
         serializer = self.get_serializer(queryset, many=True)
         return Response(serializer.data, status=status.HTTP_200_OK)
+
+
+class ProjectGetControlList(generics.ListAPIView):
+    queryset = Project.objects.all()
+    lookup_url_kwarg = "project_id"
+    serializer_class = ProjectControlListSerializer
+    filter_backends = (filters.DjangoFilterBackend,)
+    pagination_class = PageNumberPagination
+
+    def list(self, request, *args, **kwargs):
+        project_id = self.kwargs["project_id"]
+        project = self.get_object()
+        queryset = self.filter_queryset(
+            project.controls.through.objects.filter(project_id=project_id).order_by(
+                "control_id"
+            )
+        )
+        page = self.paginate_queryset(queryset)
```

Comment:
Also if you want the filters to work you'd need to specify a FilterSet class or filter fields and then add:
```python
    ...
    filtere_qs = self.filter_queryset(queryset)
    page = ...
```
Similar to the comment above, we can improve this later.